packages/lodestar/src/chain/validation/aggregateAndProof.ts

```diff
@@ -81,7 +81,7 @@ export async function validateAggregateAttestation(
   let attestationPreState;
   try {
     // the target state, advanced to the attestation slot
-    attestationPreState = await chain.regen.getBlockSlotState(attestation.data.target.root, attestation.data.slot);
+    attestationPreState = await chain.regen.getCheckpointState(attestation.data.target);
```

Comment:
maybe should be called `targetState` or `attestationTargetState`

Reply:
It's used in many places I can rename in a new PR